packages/fast-usdc/test/cli/transfer.test.ts

```diff
@@ -63,14 +64,20 @@ test('Transfer registers the noble forwarding account if it does not exist', asy
   };
   const out = mockOut();
   const file = mockFile(path, JSON.stringify(config));
-  const agoricSettlementAccount = 'agoric123456';
+  const agoricSettlementAccount =
+    'agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek';
   const settlementAccountVstoragePath = 'published.fastUsdc.settlementAccount';
   const vstorageMock = makeVstorageMock({
     [settlementAccountVstoragePath]: agoricSettlementAccount,
   });
   const amount = '150';
   const destination = 'dydx1234';
-  const nobleFwdAccountQuery = `${nobleApi}/noble/forwarding/v1/address/${nobleToAgoricChannel}/${agoricSettlementAccount}${encodeURIComponent('?EUD=')}${destination}/`;
+  const nobleFwdAccountQuery = `${nobleApi}/noble/forwarding/v1/address/${nobleToAgoricChannel}/${encodeAddressHook(
+    agoricSettlementAccount,
+    {
+      EUD: destination,
+    },
```

Comment:
style nit:

```suggestion
    {EUD: destination },
```

or better:

```suggestion
    { EUD },
```

Reply:
Surprised the linter isn't doing this for us, but good call out. Amend(ed|ing)